docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -518,6 +520,162 @@ In order to see the current maintenance state on a node:
 
   # /usr/local/pf/bin/cluster/maintenance
 
+'''
+
+Maintenance and Operation
+--------------------------
+
+Shutting down a PacketFence Active/Active cluster of tree nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to shutdown the cluster in a proper way, here is the steps you need to follow.
+We are here assuming that we are on a cluster of tree nodes, on Centos7.
+
+We will shutdown the nodes, one by one, starting by one of the "Slaves" and end with the "Master".
+
+Find the Master Server: ::
+* Type *ip a* on each nodes, and look for the one which hold the VIP (the /32 address).
+
+WARNING: *YOU TAKE THE RISK OF LOOSING SOME DATA IF YOU DO NOT IDENTIFY MASTER AND SLAVES NODES* 
+ 
+Shutdown the "Slaves": ::
+* Logon with an ssh terminal on the master and make a permanent ping on both slave.
+* On the first slave, execute: *shutdown -h now*.
+* Once the first slave down, do the same with the second slave.
+
+Shutdown the "Master": ::
+* On the master, execute: *shutdown -h now*.
+
+Bringing up a PacketFence Active/Active cluster of tree nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We want to bring up the cluster, as the same status it was before the shutdown, assuming we want to have the Master and the Slaves well synchronised.
```

Comment:
synchronized

Reply:
You can say its recommended to start them back up in the order that they were shutdown but its not a necessity.

Whether or not you follow the same order will not mean things won't work, it may just take a bit more time for the cluster to get back into a working state